platform/migration/platform-migration/src/test/java/org/codice/ddf/configuration/migration/AbstractMigrationReportSupport.java

```diff
@@ -21,9 +21,7 @@
 import org.junit.Before;
 
 /** Base class for report-type test cases which handles setup of various messages. */
-@SuppressWarnings(
-    "squid:S2187" /* base class for report-type test cases which handles setup of various messages */)
-public class AbstractMigrationReportTest extends AbstractMigrationTest {
+public class AbstractMigrationReportSupport extends AbstractMigrationSupport {
```

Comment:
🤔  I think that rule is bogus to start with.

Reply:
I think it's a slightly naive rule. For example, I doubt they have one that checks files with the suffix "Spec" for spock tests. However, it's not the worst idea in the world that your files that match a test pattern for JUnit are actually tests.

Reply:
If you read the explanation, they say that developers might get confused that there are no tests in the class! Really! The name was pretty straight forward and indicative it was an abstract class and not a test class.